cmd/fmt_test.go

```diff
@@ -835,14 +836,30 @@ q := all([true, false])
 %ROOT%/policy.rego:3: rego_type_error: deprecated built-in function calls in expression: any
 %ROOT%/policy.rego:6: rego_type_error: deprecated built-in function calls in expression: all`,
 		},
+		{
+			note:         "v1 module",
+			v1Compatible: true,
+			input: `package test
+
+p contains x if {
+	some x in ["a", "b", "c"]
+}`,
+			expected: `package test
+
+import rego.v1
+
+p contains x if {
+	some x in ["a", "b", "c"]
+}
+`,
```

Comment:
This test case illustrates a scenario that can be solved with setting the `--v1-compatible` flag. This flag is however hidden, so not ideal.

converting v0->v1 is likely the more common use case, but we can't guarantee this won't happen; and will probably become more likely as time pass ..